torch/csrc/api/src/nn/modules/loss.cpp

```diff
@@ -49,5 +49,18 @@ Tensor CosineEmbeddingLossImpl::forward(
   return F::cosine_embedding_loss(input1, input2, target, options);
 }
 
+// ============================================================================
+
+SmoothL1LossImpl::SmoothL1LossImpl(
+    const torch::nn::SmoothL1LossOptions& options_) : options(options_) {}
+
+void SmoothL1LossImpl::pretty_print(std::ostream& stream) const {
+  stream << "torch::nn::SmoothL1Loss";
```

Comment:
Thanks for the catch about `SmoothL1Loss` pretty printing! I checked the Python version and it seems to print `SmoothL1Loss()`, and I think we can also print the parentheses here (`L1loss` is a bad example and needs to be fixed in this regard :(

It would also be awesome to add a test for this pretty_print string, similar to all other modules :D